src/app/todolist/todolist.component.ts

```diff
@@ -1,27 +1,26 @@
 import { Component, OnInit, Input } from '@angular/core';
-import { DEFAULT_TODO } from './../mock-todolist';
-import { Todo } from './../todo';
+import { Observable } from 'rxjs';
+import { TodolistService } from './service/todolist.service';
 
 @Component({
   selector: 'app-todolist',
   templateUrl: './todolist.component.html',
   styleUrls: ['./todolist.component.css'],
 })
 export class TodolistComponent implements OnInit {
-  todoList = DEFAULT_TODO;
+  todoList$: Observable<[]>;
```

Comment:
- ここTodoのリストにしないとコンパイルエラー？

```suggestion
  todoList$: Observable<Todo[]>;
```

Reply:
動いてたのでエラー出てるの気付いてませんでした、、修正しました。